packages/grafana-perf-tests/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "typescript",
```

```suggestion
  "name": "@grafana/perf-tests",
```

As a public package this will try to publish as `typescript` and fail.

---

private'd  - this does not need to be published

---

Also if this needs to be a public package changes will need to happen [here](https://github.com/grafana/build-pipeline/blob/ff87b926989af25832d278a84538571898515940/pkg/tools/npm/npm.go#L26-L34).

---

now that I am thinking of it.. it might be actually easier if we published this package with ready-to-go scripts! but that can be a next step :) lets keep it private in this PR